test_unstructured/documents/test_elements.py

```diff
@@ -383,9 +374,14 @@ def and_it_serializes_a_data_source_sub_object_to_a_dict_when_it_is_present(self
         }
 
     def and_it_serializes_an_orig_elements_sub_object_to_base64_when_it_is_present(self):
+        elements = [Title("Lorem"), Text("Lorem Ipsum")]
+
+        for element in elements:
+            element.id_to_hash()
```

Comment:
I think it's better than doing this:
```
elements = assign_hash_ids(elements)
```
which has more logic underneath, and would nee to be imported

Reply:
Yeah, this one is borderline I think. We should definitely not use `documents.elements.assign_hash_ids()` because this is not its intended use and is not under test here.

But it might not hurt to add a test-utility function to `test_unstructured.unit_utils` to do this. Unfortunately the name that really makes sense to me for that is `assign_hash_ids()`, which could be a little confusing, but still, I think that's probably worthwhile because this same code passage appears a dozen times in this PR and maybe more in future. Then instead of three new lines each time it would just be:
```python
elements = assign_hash_ids([Title("Lorem"), Text("Lorem Ipsum")]
```

I think that's worth doing. We can adjust the name if we really don't like it. `using_hash_ids()` could be a good name, or maybe `with_hash_ids()`.

Or we might prefer to adjust the name of the original to include the fact that it's mapping metadata ids. Maybe something like `assign_and_map_hash_ids()`.